### DISCUSSION TITLE
Optimizations

### DISCUSSION THREAD

**Discussion by Crazyloon** [ISpaikI/Enshrouded-Server-Manager#17] [Ideas]
Let's explore the idea of doing a single server install, then replacing the settings for the server based on the selected profile, instead of requiring an install of the server for each profile directory

**Comment by ISpaikI** [comment]
thats something i thought about too. you would have to install one server that is no server and copy over all files to the other folders of active profiles by update. means not used space is lost, therefore the update of more than one server is faster.
